i8_terminal/commands/screen/screen_search.py

```diff
@@ -0,0 +1,110 @@
+
+from i8_terminal.commands.screen import screen
+from i8_terminal.common.cli import pass_command
+from i8_terminal.common.layout import df2Table
+from i8_terminal.common.metrics import (
+    get_current_metrics_df,
+    get_metric_info,
+    prepare_current_metrics_formatted_df,
+)
+from i8_terminal.common.utils import export_data, export_to_html
+from i8_terminal.config import APP_SETTINGS
+from i8_terminal.types.metric_identifier_param_type import MetricIdentifierParamType
+from i8_terminal.types.metric_view_param_type import MetricViewParamType
+from i8_terminal.types.screening_condition_param_type import ScreeningConditionParamType
+from i8_terminal.types.sort_order_param_type import SortOrderParamType
+
+
+def prepare_screen_df(
+    conditions: List[str], metrics: str, sort_by: Optional[str], sort_order: Optional[str]
+) -> Optional[pd.DataFrame]:
+    for index, condition in enumerate(conditions):
+        condition_parts = condition.split(":")
+        metric = condition_parts[0]
+        metric_parts = metric.split(".")
+        if len(metric_parts) == 1:
+            metric_default_period_type = get_metric_info(metric_parts[0])["default_period_type"]
+            metric = f"{metric}{'.mrq' if metric_default_period_type == 'Q' else '.mry' if metric_default_period_type == 'FY' else ''}"
+            conditions[index] = f"{metric}:{condition_parts[1]}"
+    tickers_list = investor8_sdk.ScreenerApi().search(
+        conditions=",".join(conditions), order_by=sort_by, order_direction=sort_order
+    )
+    screen_df = get_current_metrics_df(",".join(tickers_list), metrics)
+    return screen_df
+
+
+@screen.command()
+@click.option(
+    "--condition",
+    "condition",
+    "-c",
+    required=True,
+    type=ScreeningConditionParamType(),
+    multiple=True,
+    help="Condition of metric.",
+)
+@click.option(
+    "--view_name", "view_name", "-v", type=MetricViewParamType(), help="Metric view name in configuration file."
+)
+@click.option(
+    "--metrics",
+    "metrics",
+    "-m",
+    type=MetricIdentifierParamType(),
+    help="Comma-separated list of daily metrics.",
+)
+@click.option("--export", "export_path", "-e", help="Filename to export the output to.")
+@click.option("--sort_by", "sort_by", "-sb", type=MetricIdentifierParamType(), help="Metric to sort the output by.")
+@click.option("--sort_order", "sort_order", "-so", type=SortOrderParamType(), help="Order to sort the output by.")
+@pass_command
+def search(
+    condition: Tuple[str],
+    view_name: Optional[str],
+    metrics: Optional[str],
+    export_path: Optional[str],
+    sort_by: Optional[str],
+    sort_order: Optional[str],
+) -> None:
+    console = Console()
+    if not metrics and not view_name:
+        console.print("The 'metrics' or 'view_name' parameter must be provided", style="yellow")
+        return
+    if view_name and metrics:
+        console.print("The 'metrics' or 'view_name' options are mutually exclusive", style="yellow")
```

Comment:
Provide a message with clear instructions for the user at the end to tell them how to fix the error. For example: "Provide a value only for one of the two."